### PR TITLE
[PBF-784] Added event handler for EMBED_CODE_CHANGED_AFTER_OOYALA_AD …

### PR DESCRIPTION
**Pull request by nagavenivn**
…event

Sand box link to Test:
http://player.ooyala.com/static/v4/sandbox/PBF-784/skin-plugin/html5-skin.js
http://player.ooyala.com/static/v4/sandbox/PBF-784/core.js

### CODE REVIEW

**Comment by mistermendez** [comment]
we also need to unsubscribe from - EMBED_CODE_CHANGED_AFTER_OOYALA_AD

File: js/controller.js [hunk 1]
```diff
@@ -286,6 +287,12 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.mb = null;
     },
 
+    onEmbedCodeChangedAfterOoyalaAd: function(event, embedCode, options) {
+      if (options) {
```
Comment (aeng7) [inline js/controller.js]: Should we also check that options is not an empty object?

Comment (nagavenivn) [inline js/controller.js]: Added the code similar to the one that is already existing in onEmbedCodeChanged ()

**Comment by nagavenivn** [comment]
@mistermendez Made changes as per your comment

**Comment by aeng7** [comment]
LGTM. @mistermendez  to confirm as well.

**Comment by mistermendez** [comment]
👍

File: js/controller.js [hunk 2]
```diff
@@ -1070,6 +1077,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.mb.unsubscribe(OO.EVENTS.ASSET_UPDATED, 'customerUi');
       this.mb.unsubscribe(OO.EVENTS.PLAYBACK_READY, 'customerUi');
       this.mb.unsubscribe(OO.EVENTS.ERROR, "customerUi");
+      this.mb.unsubscribe(OO.EVENTS.SET_EMBED_CODE_AFTER_OOYALA_AD, 'customerUi');
```
Comment (mistermendez) [inline js/controller.js]: should `SET_EMBED_CODE_AFTER_OOYALA_AD` be `EMBED_CODE_CHANGED_AFTER_OOYALA_AD` here?

**Pull request merged**